src/WebViewTypes.ts

```diff
@@ -424,6 +425,11 @@ export interface IOSWebViewProps extends WebViewSharedProps {
    * @platform ios
    */
   keyboardDisplayRequiresUserAction?: boolean;
+
+  /**
+   * Function that is invoked when the WebKit WebView content process gets terminated.
+   */
+  onContentProcessDidTerminate: (event: WebViewErrorEvent) => void;
```

Comment:
I don't think this should be a WebViewErrorEvent, please create another type

Reply:
Thanks for looking at this!

I added a WebViewTerminatedEvent type. Not sure if I have done it how it is supposed to be, just let me know.

Reply:
This is fine for me :)